cluster-up/cluster/kind-1.22-sriov/sriov-components/sriov_components.sh

```diff
@@ -106,8 +108,12 @@ function sriov_components::wait_allocatable_resource() {
 }
 
 function sriov_components::deploy_multus() {
-  echo 'Deploying Multus'
-  sed "s#nfvpe/multus#quay.io/kubevirtci/multus#" "$MULTUS_MANIFEST" | _kubectl apply -f -
+  _kubectl kustomize "$KUSTOMIZE_MULTUS_DIR" > "$MULTUS_MANIFEST"
+
+  echo "Deploying Multus:"
+  cat "$MULTUS_MANIFEST"
```

Comment:
I don't think we need it, it makes the log / deploy have too much info

Reply:
I still think it is not needed, once it works it works, this code isn't breathing but making the log yet more heavy forever.

The fact it is heavy already doesn't mean we can add to it, we just need to make it lighter and to show only the important stuff.

Reply:
Let's have all the bits at the SRIOV lanes we are talking about something at it is show at startup once just before hours of tests, that's not much noise. Real noise is when we end up printing a line every 5 mintues or so .